### DISCUSSION TITLE
lighthouse10.0

### DISCUSSION THREAD

**Discussion by gk-siva** [GoogleChrome/lighthouse#14797] [Q&A]
lighthouse10.0 is available in chrome 110?

**Comment by adamraine** [comment]
No, it will be available in the DevTools of Chrome 112.

**Reply by gk-siva** [reply]
chrome beta 112 version now available?

**Reply by adamraine** [reply]
Not yet, please visit https://chromiumdash.appspot.com/schedule for more information
